scripts/celery_beat.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
 printf "celery-beat" > /tmp/container-role
 
-set -euo pipefail
+set -eo pipefail
 
 ./wait_for_db.sh
 ./wait_for_redis.sh
 
+if [[ "${AUTO_MAINTENANCE_MODE}" == "true" ]]; then
+    python manage.py maintenance_mode on
+fi
+
 python manage.py migrate --noinput
 python manage.py compilemessages -v 0
 python manage.py load_redis_index
 
+if [[ "${AUTO_MAINTENANCE_MODE}" == "true" ]]; then
```

Comment:
Do you mean false here?

Reply:
No it should be true, so the maintenance mode gets disabled automatically when the migrations ran successfully

Reply:
we can split the behavior to separate variables